server/src/main/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldMapper.java

```diff
@@ -85,28 +94,45 @@ public String typeName() {
         }
 
         @Override
-        public DocValueFormat docValueFormat(String format, ZoneId timeZone) {
-            throw new UnsupportedOperationException(ERROR_MESSAGE_7X);
+        public Query existsQuery(SearchExecutionContext context) {
+            throw new IllegalArgumentException("[sparse_vector] fields do not support [exists] queries");
```

Comment:
Why did we decide to move from `UnsupportedOperationException` to `IllegalArgumentException`? Both are fine, just curious.

Reply:
It's not really a change in the exception, as the method is different. Keep in mind that I copied / pasted `RankFeaturesFieldMapper`, so there are a lot of changes and the diff is misleading.